pmoves/services/channel-monitor/channel_monitor/monitor.py

```
@@ -448,6 +448,31 @@ def _extract_channel_id_from_url(url: Optional[str]) -> Optional[str]:
     return None
 
 
+def _normalize_channel_url(url: str) -> str:
+    """Append `/videos` tab to YouTube channel URLs so yt-dlp enumerates the
+    video list instead of returning tab metadata entries. Leaves playlists,
+    watch URLs, and non-YouTube URLs unchanged."""
+    try:
+        parsed = urlparse(url)
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)